### PR TITLE
Add alignment and filling examples in 1.2 Formatted print

### PR DESCRIPTION
**Pull request by dopin**
Add examples alignment and filling in the sample code of 1.2 Formatted print.

### CODE REVIEW

**Comment by dopin** [comment]
I'm not a native English speaker so please correct my English. :bow:

File: examples/hello/print/print.rs [hunk 1]
```diff
@@ -19,6 +19,18 @@ fn main() {
     // Special formatting can be specified after a `:`.
     println!("{} of {:b} people know binary, the other half don't", 1, 2);
 
+    // As can right-aligned in a specified width.
+    // This will output "     1". 5 white spaces and "1".
+    println!("{number:>width$}",
+             number=1,
+             width=6);
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: Can this be all one one line

File: examples/hello/print/print.rs [hunk 2]
```diff
@@ -19,6 +19,18 @@ fn main() {
     // Special formatting can be specified after a `:`.
     println!("{} of {:b} people know binary, the other half don't", 1, 2);
 
+    // As can right-aligned in a specified width.
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: "You can right-align text with a specified width"

File: examples/hello/print/print.rs [hunk 3]
```diff
@@ -19,6 +19,18 @@ fn main() {
...
+    // This will output "     1". 5 white spaces and "1".
+    println!("{number:>width$}",
+             number=1,
+             width=6);
+
+    // As can filled with 0.
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: You can pad numbers with extra zeroes.

File: examples/hello/print/print.rs [hunk 4]
```diff
@@ -19,6 +19,18 @@ fn main() {
     // Special formatting can be specified after a `:`.
     println!("{} of {:b} people know binary, the other half don't", 1, 2);
 
+    // As can right-aligned in a specified width.
+    // This will output "     1". 5 white spaces and "1".
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: These should also be on the same line as 22, not one sentence per line

File: examples/hello/print/print.rs [hunk 5]
```diff
@@ -19,6 +19,18 @@ fn main() {
...
+    println!("{number:>width$}",
+             number=1,
+             width=6);
+
+    // As can filled with 0.
+    // This will output "000001".
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: These should also be on the same line as 28, not one sentence per line

File: examples/hello/print/print.rs [hunk 6]
```diff
@@ -19,6 +19,18 @@ fn main() {
...
+    println!("{number:>0width$}",
+             number=1,
+             width=6);
```
Comment (steveklabnik) [inline examples/hello/print/print.rs]: same, this should be on one line

**Comment by steveklabnik** [comment]
Thanks for this! No worries, it's understandable, but I made it a little more idiomatic :)

File: examples/hello/print/print.rs (hunk 2 above)
Comment (mdinger) [inline examples/hello/print/print.rs]: Also `s/As can/Can be/` and `s/in a/with a/` would probably be okay. They're both pretty similar.

File: examples/hello/print/print.rs (hunk 4 above)
Comment (mdinger) [inline examples/hello/print/print.rs]: "5 white spaces and **a** "1""

**Comment by dopin** [comment]
Thank you very much for correcting, @steveklabnik and @mdinger. :joy: 
I've fixed and squashed. Could you check it again?

**Comment by steveklabnik** [comment]
It looks great, except for the wrapping: the build failed because the lines are too long.

**Comment by dopin** [comment]
my bad... I'm sorry. I should have read the guide.
[Lines should contain a maximum of 79 characters.](https://github.com/rust-lang/rust-by-example/blob/master/CONTRIBUTING.md#markdown-md).
Fixed!!

**Comment by steveklabnik** [comment]
Thank sos much!

**Comment by dopin** [comment]
Thank you! :laughing:

**Pull request merged**